### ISSUE TITLE
Incompatible with node v4.0.0 and node red v 0.11.1 on windows?

### ISSUE THREAD

**Issue by amitmangalvedkar** [ADVANTECH-Corp/node-red-contrib-alljoyn#1]
I have been trying to install node-red-contrib-alljoyn on my windows box, but it always gives the error 
npm WARN engine node-alljoyn@0.1.1: wanted: {"node":"^0.10.35"} (current: {"node
":"4.0.0","npm":"2.14.3"})
Does the node work on the above stack

**Comment by daniel6096** [comment]
No. It only works on node 0.10.x now, not test on node v4.0.0 yet. You may modify the package.json to verify.
https://github.com/ADVANTECH-Corp/node-alljoyn/blob/master/package.json

Furthermore, I only test on Yocto Linux. Not sure if it works on Windows.
I guess it does not work, either. You may need to add some library paths, link settings & dependencies.

**Comment by amitmangalvedkar** [comment]
Thanks for the quick response.

Thanks and Regards
Amit M Mangalvedkar
